core/modules/search/search.admin.inc

```diff
@@ -105,66 +119,83 @@ function search_admin_settings($form, &$form_state) {
     '#value' => t('Save configuration')
   );
 
-  // Indexing settings:
-  $form['indexing'] = array(
-    '#type' => 'fieldset',
-    '#title' => t('Indexing'),
-    '#description' => t('<p><em>The search index is not cleared immediately when invalidated, instead it will be systematically updated during cron runs. Searching will continue to work but new content won\'t be indexed until all existing content has been re-indexed.</em></p>'),
-  );
-
-
-  // Collect some stats
-  $remaining = 0;
-  $total = 0;
-  foreach ($config->get('search_active_modules') as $module) {
-    if ($status = module_invoke($module, 'search_status')) {
```

Comment:
Hm, looks like this removes `hook_search_status()` entirely. Can we maintain this hook but make it backwards-compatible somehow?